Source/init.cpp

```diff
@@ -94,7 +94,7 @@ HANDLE init_test_access(char *mpq_path, char *mpq_name, char *reg_loc, int dwPri
 
 	for (int i = 0; i < 2; i++) {
 		snprintf(mpq_path, MAX_PATH, "%s%s", Buffer[i], mpq_name);
-#ifndef __SWITCH__
+#if !defined(__SWITCH__) || !defined(__3DS__)

```

Comment:
This `||` should be `&&`
If this wasn't causing any issues, perhaps this is not a necessary hack on the 3DS?

Reply:
Due to the similarities between the switch and 3ds, i like to take it as reference. I had some issues running the .cia. This is a leftover from me desperately trying. I'll have it removed.